packages/test/local-server-stress-tests/src/test/localServerStress.spec.ts

```diff
@@ -29,13 +29,9 @@ describe("Local Server Stress", () => {
 		saveFailures,
 		// saveSuccesses,
 		skip: [
-			...[15], // 0x54e (IntervalCollection)
-			...[39], // 0xa6f (merge-tree PropertiesManager)
```

Comment:
@anthony-murphy do we want to save these seeds somewhere as repro's?  I had opened a task about this I think but hadn't gotten around to it.  Leaving this comment here as a bookmark to find these in the future if so desired.

Reply:
Not sure how to get around the issue with not wanting the output format to be a supported long-term schema though.

Reply:
bugs with the replay file would be nice

Reply:
Opened AB#39070 with both the failures from before this change, and also the failures with this change.